nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -485,98 +485,137 @@ It is preferable to request upstream/downstream systems to switch to https://cwi
+As noted, the nodes communicate with the Cluster Coordinator via heartbeats. When a Cluster Coordinator is elected, it updates
+a well-known ZNode in Apache ZooKeeper with its connection information so that nodes understand where to send heartbeats. If one
+of the nodes goes down, the other nodes in the cluster will not automatically pick up the load of the missing node. It is possible
+for the DFM to configure the dataflow for failover contingencies; however, this is dependent on the dataflow design and does not
+happen automatically.
+
+When the DFM makes changes to the dataflow, the node that receives the request to change the flow communicates those changes to all
+nodes and waits for each node to respond, indicating that it has made the change on its local flow.
 
-When the DFM makes changes to the dataflow, the NCM communicates those changes to the nodes and waits for each node to respond, indicating that it has made the change on its local flow. If the DFM wants to make another change, the NCM will only allow this to happen once all the nodes have acknowledged that they've implemented the last change. This is a safeguard to ensure that all the nodes in the cluster have the correct and up-to-date flow.
 
 *Dealing with Disconnected Nodes* +
 
-A DFM may manually disconnect a node from the cluster. But if a node becomes disconnected for any other reason (such as due to lack of heartbeat), the NCM will show a bulletin on the User Interface, and the DFM will not be able to make any changes to the dataflow until the issue of the disconnected node is resolved. The DFM or the Administrator will need to troubleshoot the issue with the node and resolve it before any new changes may be made to the dataflow. However, it is worth noting that just because a node is disconnected does not mean that it is not working; it just means that the NCM cannot communicate with the node.
+A DFM may manually disconnect a node from the cluster. But if a node becomes disconnected for any other reason (such as due to lack of heartbeat),
+the Cluster Coordinator will show a bulletin on the User Interface. The DFM will not be able to make any changes to the dataflow until the issue
+of the disconnected node is resolved. The DFM or the Administrator will need to troubleshoot the issue with the node and resolve it before any
+new changes may be made to the dataflow. However, it is worth noting that just because a node is disconnected does not mean that it is not working;
+this may happen for a few reasons, including that the node is unable to communicate with the Cluster Coordinator due to network problems.
```

Comment:
After the DFM marks a node as disconnected, they have the option of removing the node from the cluster. In that case, they can make changes to the flow. Will update the guide to explain this as well.